app/controllers/password_reset_controller.rb

```diff
@@ -0,0 +1,28 @@
+class PasswordResetController < ApplicationController
+
+  def create
+    user = User.find_by(email: params[:email])
+    user.send_password_reset if user
+    redirect_to login_url, :notice => "Email sent with password reset instructions."
+  end
+
+  def edit
+    @user = User.find_by(password_reset_token: params[:id])
+    if @user.password_reset_token_generated_at < 2.hours.ago
```

Comment:
@rishabhguptarishi 
1. what if user not found
2. don't hard code 2 hours here. move it to config variables maintained by figaro